src/components/UserChip/UserChip.tsx

```diff
@@ -1,82 +1,21 @@
-import Avatar from "@material-ui/core/Avatar";
-import Chip from "@material-ui/core/Chip";
-import ClickAwayListener from "@material-ui/core/ClickAwayListener";
 import FormControlLabel from "@material-ui/core/FormControlLabel";
-import Grow from "@material-ui/core/Grow";
-import Hidden from "@material-ui/core/Hidden";
-import MenuItem from "@material-ui/core/MenuItem";
-import Menu from "@material-ui/core/MenuList";
-import Paper from "@material-ui/core/Paper";
-import Popper from "@material-ui/core/Popper";
 import Switch from "@material-ui/core/Switch";
 import { User } from "@saleor/fragments/types/User";
-import ArrowDropdown from "@saleor/icons/ArrowDropdown";
-import { makeStyles } from "@saleor/macaw-ui";
+import {
+  makeStyles,
+  UserChip as MacawUserChip,
```

Comment:
```suggestion
  UserChip as UserChipGroup,
```
Perhaps this should be the original name of this component as it can only be used in conjunction with UserChipMenuItems.

Reply:
Makes sense. If you have no other things to change, I'd like to merge this as it is for now and I'll rename it in the next PR with macaw's API improvements.